Rewrite the Readme for EFCore (#13)
DevExpress-Examples/XAF_how-to-connect-different-data-models-to-several-databases-within-a-single-application: Readme.md

[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# How to connect different data models to several databases within a single application with Entity Framework Core

> This example demonstrates how to create custom XAF modules with custom business objects and logic that would work with separate databases. These modules do not depend on each other and thus can be reused in other applications as a whole.  

Reviewer: 1. `This example demonstrates how to create custom XAF modules with custom business objects and logic that would work with separate databases.`

Can you change "that would work with separate databases" to "that work with separate databases"?

2. `These modules do not depend on each other and thus can be reused in other applications as a whole.`

By "as a whole" do you mean "independently"?

Author: I applied both suggestions, thank you!

Reviewer: Thank you!

> The [EFCore](./CS/EFCore/) example solution illustrates the following documentation article: [How to: Use Multiple Data Models Connected to Different Databases in Entity Framework Core](https://docs.devexpress.com/eXpressAppFramework/404322).

Reviewer: ```suggestion
This [EFCore](./CS/EFCore/) example solution is based on the following tutorial: [How to: Use Multiple Data Models Connected to Different Databases in Entity Framework Core](https://docs.devexpress.com/eXpressAppFramework/404322).
```
